app/models/cohort.rb

```diff
@@ -0,0 +1,8 @@
+class Cohort < ApplicationRecord
+  belongs_to :track
+
+  has_many :cohort_memberships, dependent: :destroy
+  has_many :memberships, through: :cohort_memberships, source: :cohort
```

Comment:
This is wrong.

I think you mean
```
has_many :members, through: :cohort_memberships, source: :users
```

(memberships -> members, cohort -> user)

Reply:
I do. Thanks